assets/multiqc_config.yml

```diff
@@ -5,11 +5,14 @@ report_section_order:
     order: -1000
   busco:
     order: -1001
-  "Arcadia-Science-hifi2genome-methods-description":
+  Samtools:
```

Comment:
is capitalization intentional here?

Reply:
yes and it is dumb, if you don't capitalize it won't make the ordering correct